common-theme/layouts/_default/prep.html

```diff
@@ -25,4 +25,16 @@
 
   {{ $soloView := resources.Get "scripts/solo-view.js" | resources.Minify }}
   <script src="{{ $soloView.RelPermalink }}" defer></script>
+  {{ with or .Params.prerequisites .Parent.Params.prerequisites }}
+    <footer class="c-page-footer">
+      <h3 class=" e-heading__4">🎟️ Prerequisites</h3>
```

Comment:
Can't see the prerequisites straight away in the footer. Suggest moving them just underneath page header description?

Reply:
Hmmmmm, not sure I agree. (Remembering this will have the label layout once merged).

I don't think we want this straight away - it's surrounding metadata, not the first thing you look for. Can you say why it should be so highly prioritised?

Reply:
One sec I'll move it to your suggestion so we can look, too